examples/tests/data_silo_plugin/main.tf

```diff
@@ -0,0 +1,62 @@
+terraform {
+  required_providers {
+    transcend = {
+      version = "0.5.0"
+      source  = "transcend.com/cli/transcend"
+    }
+  }
+}
+
+provider "transcend" {
+  url = "https://api.dev.trancsend.com/"
+}
+
+variable "schedule_frequency" {
+  type = string
+  default = "3000"
+}
+
+variable "schedule_start_at" {
+  type = string
+  default = "2022-08-16T07:00:00.000Z"
+}
+
+variable "enabled" {
+  type = bool
+  default = true
+}
+
+resource "transcend_data_silo" "gradle" {
+  type = "gradle"
+}
+
+data "transcend_data_silo_plugin" "gradlePlugin" {
```

Comment:
I wonder if we could eliminate the `plugin_id` field on `transcend_data_silo_plugin` and have this lookup happen inside the resource?

Reply:
I did a read first prior to creating which seems to work in this case, so you won't have to provide a pluginID anymore in order to create the resource